assets/src/js/admin/reports/app/pages/overview-page/index.js

```diff
@@ -2,231 +2,185 @@
 // Pages use the Grid component to establish a
 // 12 column grid for content to exist in
 
-import Grid from '../../../components/grid'
-import Card from '../../../components/card'
-import Chart from '../../../components/chart'
-import RESTChart from '../../../components/rest-chart'
-import MiniChart from '../../../components/mini-chart'
-import List from '../../../components/list'
-import RESTList from '../../../components/rest-list'
-import LocationItem from '../../../components/location-item'
-import DonationItem from '../../../components/donation-item'
-import DonorItem from '../../../components/donor-item'
+import Grid from '../../../components/grid';
+import Card from '../../../components/card';
+import RESTChart from '../../../components/rest-chart';
```

Comment:
@henryholtgeerts did you create any issue to create `@givewp/components` workspace to prevent writing  `import` statements like this and make code DRYer.
you can also review `givewp-campaigns` or `giveio`.

Reply:
Yes - there is an open issue for this, and some work has been done to this end https://github.com/impress-org/givewp/issues/4349